### DISCUSSION TITLE
https://nolebase-template-ivory.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%B5%8B%E8%AF%95%E6%96%87%E6%A1%A3.html

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#33] [Announcements]
# https://nolebase-template-ivory.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%B5%8B%E8%AF%95%E6%96%87%E6%A1%A3.html

记录回忆，知识和畅想的地方

https://nolebase-template-ivory.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%B5%8B%E8%AF%95%E6%96%87%E6%A1%A3.html

**Comment by Stonefox-R** [comment]
test
